internal/kafka/internal/services/clusters.go

```diff
@@ -712,6 +712,7 @@ type KafkaStreamingUnitCountPerCluster struct {
 	CloudProvider string
 	MaxUnits      int32
 	Status        string
+	ClusterType   string
```

Comment:
How does this impact dynamic scaling evaluation? Are enterprise clusters explicitly excluded there?

Reply:
edit: I see code in scaleup and scaledown mgr has been modified to take it into account

Reply:
it should not affect it